tests/Parse/IniTest.php

```diff
@@ -244,7 +244,7 @@ public function testMultilinesValues()
         $vars = [
             'var' => "\\Test\\Path\\",
             'editorContent' =>
-'<p>Some
+        '<p>Some
```

Comment:
What's the reason for this change?

Reply:
PHPCS sees the above as incorrect indentation. I've formatted the string content there to match the original spacing but also keep the correct indentation.

https://github.com/octobercms/library/pull/428/commits/c487ef37cbf84844c92aa46fc296ff69672ba93a